matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/room/send/LocalEchoEventFactory.kt

```diff
@@ -795,16 +796,21 @@ internal class LocalEchoEventFactory @Inject constructor(
         }
     }
      */
-    fun createRedactEvent(roomId: String, eventId: String, reason: String?, additionalContent: Content? = null): Event {
+    fun createRedactEvent(roomId: String, eventId: String, reason: String?, withRelations: List<String>? = null, additionalContent: Content? = null): Event {
         val localId = LocalEcho.createLocalEchoId()
+        val content = if (reason != null || withRelations != null) {
+            EventRedactBody(reason, withRelations, withRelations).toContent().plus(additionalContent.orEmpty())
```

Comment:
Same remark, I think it would be clearer to have named arguments for `EventRedactBody`.

Reply:
https://github.com/vector-im/element-android/pull/7988/commits/52d0b826f494e2c19e2b2b8818f853ac0b03596f